THIRD-PARTY-LICENSES

```diff
@@ -27,3 +27,10 @@ Authors: okdshin
 License: MIT License
 
 -------------------------------------------------------------------------
+
+Package: cpputil
+Authors: Will Zhang
+Source: https://github.com/willzhang4a58/cpputil
```

Comment:
Nice to see. Big thanks for dealing with licensing and big thanks to @willzhang4a58 for the code.